client/javascripts/app.js

```diff
@@ -1,4 +1,5 @@
 import riot from 'riot'
 import './components/**/*.tag'
+import './tags/*.tag'
```

Comment:
Why don't you just use /components/

Reply:
I don't know it's not really a component to me, but I can do that